### ISSUE TITLE
Add-on: Social Sharing Buttons

### ISSUE THREAD

**Issue by archetyped** [archetyped/simple-lightbox#206]
This add-on adds social sharing buttons to the lightbox.

Add-ons are developed based on demand.  Vote for this plugin by clicking the 👍 in the corner of this post and provide your feedback on this functionality and how you would like it to work by commenting below.

**Comment by dadra** [comment]
Social sharing buttons would be a great addition to a great lightbox plugin!  In my ideal universe there would be an option so the various buttons could be activated individually to create custom combinations.  Definitely looking forward to this!  :-)

**Comment by lew1s** [comment]
i am looking for this option as well. Any news?

**Comment by zephyr88** [comment]
Indeed, it's 2016 already, a great feature in this world of social media

**Comment by tonyassante** [comment]
is possible insert ads on every single photo in light box? Tony Assante -Costruzione Siti WEB 

    Il Giovedì 5 Maggio 2016 12:02, zephyr88 <notifications@github.com> ha scritto:
 

 Indeed, it's 2016 already, a great feature in this world of social media—
